manage.py

```diff
@@ -22,24 +18,6 @@ def cli():
 TIMESCALE_SQL_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'admin', 'timescale')
 
 
-@cli.command()
-@click.option("--host", "-h", default="0.0.0.0", show_default=True)
-@click.option("--port", "-p", default=8080, show_default=True)
-@click.option("--debug", "-d", is_flag=True,
-              help="Turns debugging mode on or off. If specified, overrides "
-                   "'DEBUG' value in the config file.")
-def runserver(host, port, debug=False):
-    application = webserver.create_app()
-    run_simple(
-        hostname=host,
-        port=port,
-        application=application,
-        use_debugger=debug,
-        use_reloader=debug,
-        processes=5
```

Comment:
This is a change compared to what was previously here. Now it's only using 1 process. @mayhem added this in https://github.com/metabrainz/listenbrainz-server/commit/610f0cba83405413b9762367d2faa1b857bedc83, but it's not clear why we needed this back in 2017. I've never seen a problem with needing more than 1 process in a development server, either for LB or AB, but it's possible that there was a clear reason for this. We can change it back if needed

Reply:
I like to have more than one process, so that images and other things don't all wait for one process. It makes the debugging a little smoother overall. The rest? 🗑️

Reply:
The `flask run` command that I added here uses werkzeug's `threaded=True` flag, which spawns a new thread for each request. This flag is incompatible with `processes`: https://github.com/pallets/werkzeug/blob/1129879c79f8415d4ea9a67d7c5ce2f2e1d28c78/src/werkzeug/serving.py#L691-L692
it seems like they effectively have the same behaviour, so it looks like this works as you want.